roles/openshift_aws/tasks/vpc.yml

```diff
@@ -47,6 +62,6 @@
     tags:
       Name: "{{ openshift_aws_vpc_name }}"
     routes:
-    - dest: 0.0.0.0/0
-      gateway_id: igw
+      - dest: 0.0.0.0/0
```

Comment:
undo this change.

Reply:
This is necessary to pass yamllint otherwise we get...
```
$ yamllint roles/openshift_aws/tasks/vpc.yml
roles/openshift_aws/tasks/vpc.yml
  65:5      error    wrong indentation: expected 6 but found 4  (indentation)
```